Nos/Models/CoreData/Event+CoreDataClass.swift

```diff
@@ -51,6 +51,8 @@ public enum EventKind: Int64, CaseIterable, Hashable {
     case mute = 10_000
     case auth = 27_235
     case longFormContent = 30_023
+    case liveEvent = 30_311
```

Comment:
We should start documenting what these events kinds are. And where to find more information about them

Reply:
I'm adding docs now, though I'm not 100% sure what all to include. I think these should be short, though, so I'm doing that.

Reply:
Done! And moved EventKind to its own file.